includes/admin/downloads/metabox.php

```diff
@@ -496,9 +497,12 @@ function edd_render_product_type_field( $post_id = 0 ) {
  * @param $post_id
  */
 function edd_render_products_field( $post_id ) {
-	$type     = edd_get_download_type( $post_id );
-	$display  = $type == 'bundle' ? '' : ' style="display:none;"';
-	$products = edd_get_bundled_products( $post_id );
+	$type             = edd_get_download_type( $post_id );
+	$display          = $type == 'bundle' ? '' : ' style="display:none;"';
+	$products         = edd_get_bundled_products( $post_id );
+	$variable_pricing = edd_has_variable_prices( $post_id );
+	$variable_display = $variable_pricing ? '' : 'display:none;';
+	$prices           = edd_get_variable_prices( $post_id );
```

Comment:
For any items that exist in `EDD_Download`, let's convert to using the class instead of the helper functions.